T/TempestRemap/build_tarballs.jl

```diff
@@ -89,6 +89,7 @@ products = [
 
 dependencies = [
     Dependency("OpenBLAS32_jll"),
+    Dependency("HDF5_jll", compat="~1.12.2"),
```

Comment:
Why does 12.2 make a difference here?

Reply:
Because when there's only `compat` we use the minimum version of the range for building, and apparently NetCDF wasn't compatible with hdf5 12.2.0, but only 12.2.2